joss_paper/paper.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'SunPy: A Python package for Solar Physics'
+tags:
+  - Python
+  - Astronomy
+  - Solar physics
+authors:
+  - name: Stuart J. Mumford
+    orcid: 0000-0003-4217-4642
+    affiliation: "1, 2"
+  - name: Nabil Freij
+  - name: Steven Christe
+  - name: Jack Ireland
+  - name: Florian Mayer
+  - name: Keith Hughitt
+  - name: Albert Y. Shih
+  - name: Daniel Ryan
+  - name: Simon Liedtke
+  - name: David Pérez-Suárez
+  - name: Pritish Chakraborty
+  - name: Vishnunarayan K I.
+  - name: Andrew Inglis
+  - name: Punyaslok Pattnaik
+  - name: Brigitta Sipocz
```

Comment:
```suggestion
  - name: Brigitta Sipőcz
    orchid: https://orcid.org/0000-0002-3713-6337
    affiliation: "DIRAC Institute, Department of Astronomy, University of Washington, Seattle, WA 98195, USA"
```

Reply:
git shortlog did not enjoy the correct spelling of your surname clearly!

Reply:
yes, I'm old school with e-mail and still do ascii, only even when writing Hungarian.